api/v3/Address.php

```diff
@@ -110,6 +114,11 @@ function _civicrm_api3_address_create_spec(&$params) {
       at once, the job \'Geocode and Parse Addresses\' can execute this task after the import)',
     'type' => CRM_Utils_Type::T_BOOLEAN,
   );
+  $params['fix_address'] = array(
+    'title' => 'Fix address',
+    'description' => 'When true, apply various fixes to the address before insert. Default true.',
+    'type' => CRM_Utils_Type::T_BOOLEAN,
```

Comment:
You can add `'api_default' => TRUE` and then you won't need the additions on L79.